service/src/main/java/com/zebrunner/reporting/service/impl/StorageServiceImpl.java

```diff
@@ -100,8 +100,7 @@ private void validateObject(BinaryObject binaryObject) {
 
     private String buildObjectKey(BinaryObject binaryObject) {
         String name = UUID.randomUUID().toString();
```

Comment:
Maybe i am confusing things but i guess we agreed that we have to keep tenancy context here